entities/src/main/java/org/odk/collect/entities/storage/Entity.kt

```diff
@@ -1,17 +1,28 @@
 package org.odk.collect.entities.storage
 
 sealed interface Entity {
-    val list: String
     val id: String
     val label: String?
     val version: Int
     val properties: List<Pair<String, String>>
     val state: State
+
+    /**
+     * The server version (from an entity list CSV) this is based on.
```

Comment:
I think this stil needs clarification to understand the difference between `version` and `trunkVersion`.